Extensions/TextInput/textinputruntimeobject.ts

```diff
@@ -500,6 +531,37 @@ namespace gdjs {
     isFocused(): boolean {
       return this._renderer.isFocused();
     }
+    isSubmitted(): boolean {
+      console.log(this._renderer.getSubmitted());
+
+      return this._renderer.getSubmitted();
```

Comment:
See other comment: I think this `isSubmitted` state should be stored at the object level.